src/components/base/animation/expand/Expand.js

```diff
@@ -2,14 +2,15 @@ import Icon from "@/components/base/icon/Icon";
 
 import styles from "./Expand.module.css";
 
-export default function Expand({ open, size = 3 }) {
+export default function Expand({ open, size = 3, src, bgColor }) {
   return (
     <Icon
       size={size}
-      bgColor="var(--blue)"
+      // bgColor="var(--blue)"
       dataCy="expand-icon"
       className={`${styles.expandicon} ${open ? styles.opened : styles.closed}`}
       alt="expand"
+      src="expand.svg"
```

Comment:
Bliver bgColor og src ikke brugt ? bgColor kunne evt. have default farve på "var(--blue)". Så der mindre chance for at det påvirker de steder hvor expand bliver brugt